### ISSUE TITLE
Bidding not possible in a51caf14

### ISSUE THREAD

**Issue by NullCase** [RoboMWM/GPAuctions#36]
Test: create auction, try to bid.
Result: Sign does not update. Bid not registered. No message to the player.
Expected: Update to sign, registered bid, message to the player.

```
18.03 12:12:34 [Server] INFO [GriefPrevention] NullCase placed a sign @ world: x3335, z-285;[ac];999
18.03 12:12:34 [Server] INFO [GPAuctions] SignChangeEvent called
18.03 12:12:34 [Server] INFO [GPAuctions] Line 1 matched sign
18.03 12:12:34 [Server] INFO [GPAuctions] Claim found
18.03 12:12:34 [Server] INFO [GPAuctions] Set starting bid to 999.0
18.03 12:12:34 [Server] INFO [GPAuctions] addAuction called
18.03 12:12:34 [Server] INFO [GPAuctions] Pre-existing auction does not exist.
18.03 12:12:34 [Server] INFO [GPAuctions] Set claim owner to null (admin claim) and cleared trustlist
18.03 12:12:34 [Server] INFO [GPAuctions] Deducted 15 bonus blocks from player ac3b0963-75bd-44d2-b6ad-12efa76985cc
18.03 12:12:34 [Server] INFO [GPAuctions] sign location save Location{world=CraftWorld{name=world},x=3335.0,y=72.0,z=-291.0,pitch=0.0,yaw=0.0}
18.03 12:12:34 [Server] INFO [GPAuctions] sign location save Location{world=CraftWorld{name=world},x=3335.0,y=72.0,z=-285.0,pitch=0.0,yaw=0.0}
18.03 12:12:34 [Server] INFO [GPAuctions] Saved auctions to file.
18.03 12:12:34 [Server] INFO [GPAuctions] Adding auction with ID 2983
18.03 12:12:34 [Server] INFO [GPAuctions] Auction started. Claim ID: 2983 endTime: 1552929154524 startingBid: 999.0
18.03 12:12:34 [Server] INFO Unknown command. Type "/help" for help.
18.03 12:12:36 [Server] INFO [GPAuctions] Sign clicked
18.03 12:12:36 [Server] INFO [GPAuctions] Is a real estate-labeled sign
18.03 12:12:36 [Server] INFO [GPAuctions] No auction found or incorrect sign used at location Location{world=CraftWorld{name=world},x=3335.0,y=72.0,z=-285.0,pitch=0.0,yaw=0.0}
```

**Comment by NullCase** [comment]
I don't know but it seems like "Only allow interaction with the primary auction sign. Closes #35" might be the thing. Not finding the primary auction sign.

**Comment by RoboMWM** [comment]
~~As per the log, a sign location is being saved twice with different coordinates... will investigate.~~ That's probably from another auction, since it saves all in the file.
